### DISCUSSION TITLE
Embedded image support?

### DISCUSSION THREAD

**Discussion by Thecactusman0** [schreibfaul1/ESP32-audioI2S#594] [Q&A] [answered]
I can see that in the code there seems to be some support for images, but I can't see anyway to actually use this functionality. It looks like it was added by someone for the project but they couldn't get it to work. Is there any way to use this function or are there no plans for it?

Thanks

**Comment by schreibfaul1** [comment, marked as answer]
Yes, some audio files have embedded graphics, so-called APIC tags. The audioI2S library jumps over the graphic at the beginning of the audio block. This generates an event that passes the position of the image on the SD card and its size. In the project [ESPuino](https://github.com/biologist79/ESPuino) this is done and the cover image is displayed on a web page.
